move-mutator/src/operators/binary.rs

```diff
@@ -85,8 +89,33 @@ impl MutationOperator for Binary {
             },
         };
 
+        let is_left_exp_zero = contains_value_zero(self.exps[0].exp.as_ref());
+        let is_right_exp_zero = contains_value_zero(self.exps[1].exp.as_ref());
+
         ops.into_iter()
             .filter(|v| cur_op != *v)
+            .filter(|v| match cur_op {
```

Comment:
In general, we could apply the same logic to MAX value for the same type like for x being u8: `x == 255` should never mutate to `x >= 255`

Reply:
Due to missing type info in the AST node, we won't be able to implement that easily.
Let's discuss it on Slack